### ISSUE TITLE
aria-roledescription cannot be applied to a div (such as slideshow component) without correct role (role="group")

### ISSUE THREAD

**Issue by jlahijani** [uikit/uikit#4735]
### UIkit Version
3.16.3

### Description

With UIkit 3.16.x, the slideshow component gets `aria-roledescription` automatically added to the div.  The [UIkit doc](https://getuikit.com/docs/slideshow#accessibility) specifically states this: 
> The slideshow has the aria-roledescription property set to carousel

Lighthouse and WAVE do not bring up any issues, but axe Devtools does bring this up as a "Serious" issue:
![2023-03-01_12-27-42](https://user-images.githubusercontent.com/4558/222257990-5ee4a345-d38f-467b-8ddd-933fb7666418.png)

It references this page as to the correct usage of `aria-roledescription`:
https://dequeuniversity.com/rules/axe/4.6/aria-roledescription?application=AxeChrome

However if we added `role="group"` to the div, it will pass.  Explanation:
https://w3c.github.io/aria-practices/#basic-carousel-elements

Perhaps the examples in the documentation should include the `role="group"` part?
